### DISCUSSION TITLE
Manual/class-Font

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#52] [Notes]
# Manual/class-Font

https://docs.unity3d.com/Manual/class-Font.html

**Comment by vertxxyz** [comment]
If you're looking for a monospaced font for an editor extension:
```csharp
EditorGUIUtility.Load("Fonts/RobotoMono/RobotoMono-Regular.ttf") as Font;
```

**Comment by docscomments** [comment]
**Comment by `@szymon112233`:**  

Be sure to separate Fallback Fonts in 'Font Names' with commas (Font 1, Font 2, Font 3). Delimiting by new lines alone will not work.
